ops-tools/docker/images/app/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM openjdk:11
+
+COPY ../../../../src /usr/src/DockerPracticeJavaApplication/
+COPY ../../../../pom.xml /usr/src/DockerPracticeJavaApplication/
+
+WORKDIR /usr/src/DockerPracticeJavaApplication
+
+RUN apt-get update && apt-get install -y maven
```

Comment:
The best practice here is to add additional cleanup command after the apt used for its index update and packages installation. Read more about it at https://docs.docker.com/develop/develop-images/dockerfile_best-practices/

> In addition, when you clean up the apt cache by removing /var/lib/apt/lists it reduces the image size, since the apt cache is not stored in a layer. Since the RUN statement starts with apt-get update, the package cache is always refreshed prior to apt-get install.